### PR TITLE
Add asset-manager

### PR DESCRIPTION
**Pull request by kentsanggds**
## What 

Add asset manager to test and integration cluster

## Reference 

https://trello.com/c/pRBFWfGx/688-asset-manager-replatforming

### CODE REVIEW

File: charts/argocd-apps/values-test.yaml [hunk 1]
```diff
@@ -579,3 +579,57 @@ applications:
...
+        valueFrom:
+          secretKeyRef:
+            name: signon-app-asset-manager
+            key: oauth_secret
+      - name: GOVUK_APP_NAME
+        value: asset-manager
+      - name: GOVUK_APP_ROOT
+        value: /var/apps/static
+      - name: GOVUK_APP_TYPE
+        value: rack
```
Comment (sengi) [inline charts/argocd-apps/values-test.yaml:614]: GOVUK_APP_TYPE doesn't do anything; we can ditch it.
```suggestion
```

File: charts/argocd-apps/values-test.yaml [hunk 2]
```diff
@@ -579,3 +579,57 @@ applications:
...
+          secretKeyRef:
+            name: authenticating-proxy-jwt-auth-secret
+            key: JWT_AUTH_SECRET
+      - name: MONGODB_URI
+        valueFrom:
+          secretKeyRef:
+            name: asset-manager-docdb
+            key: MONGODB_URI
+      - name: PORT
+        value: "3000"
```
Comment (sengi) [inline charts/argocd-apps/values-test.yaml:628]: PORT is now set for you in the govuk-rails-app chart so this is no longer needed.
```suggestion
```

File: charts/argocd-apps/values-test.yaml [hunk 3]
```diff
@@ -579,3 +579,57 @@ applications:
...
+      enabled: true
+      annotations:
+        alb.ingress.kubernetes.io/load-balancer-name: asset-manager
+      hosts:
+      - name: asset-manager.eks.test.govuk.digital
+    replicaCount: 1
+    workerReplicaCount: 1
+    extraEnv:
+      - name: ASSET_MANAGER_CLAMSCAN_PATH
+        value: /bin/true
```
Comment (sengi) [inline charts/argocd-apps/values-test.yaml:598]: This definitely needs a TODO!

**Review by sengi — APPROVED** [review]

**Comment by kentsanggds** [comment]
OK, I'll update the yaml

**Pull request merged**